csv-validator-ui/src/main/scala/uk/gov/nationalarchives/csv/validator/ui/CsvValidatorUi.scala

```diff
@@ -64,14 +64,14 @@ object CsvValidatorUi extends SimpleSwingApplication {
 
   private def changeHeightOfOutputPane(heightDiff: Int): Unit = {
     // need to normalise the height change by converting it to "rows"
-    val heightChangeAsRows = Math.ceil(heightDiff / 50f).toInt // 30 is just an arbitrary number that seemed to work fine
-    val newHeight = if((txtArReport.rows + heightChangeAsRows) <= 1) txtArReport.rows + 1
-    else if((txtArReport.rows + heightChangeAsRows) > 40) 40
+    val heightChangeAsRows = Math.ceil(heightDiff / 30f).toInt // 30 is just an arbitrary number that seemed to work fine
```

Comment:
this seem to be quite a long winded way of just doing `heightDiff / 30` 🤔

Reply:
Ah, yes, I did this so that it rounds up to the nearest whole number because let's say there is an increase/decrease of 25 for example, 25/30 = 0  (because you're dividing Ints) and then it won't change the height. So I need to make 30 a float, so that I can get `0.8333333`, round it up using Ceil, to 1.0 and then convert the result to an Int.

Reply:
Got it. Fair enough.